lib/gui/pages/main/controllers/flash.js

```diff
@@ -117,23 +117,12 @@ module.exports = function(
    */
   this.getProgressButtonLabel = () => {
     const currentFlashState = flashState.getFlashState();
```

Comment:
This is very minor, but can we move this line above `return progressStatus.makeSubtitle(currentFlashState);` so that we only get the current flash state if we need to (e.g. if we're really flashing)